app/views/content/life-as-a-teacher/teaching-as-a-career/early-career-teachers.md

```diff
@@ -2,34 +2,29 @@
 title: "Early career teachers (ECTs)"
 heading: "Early career teachers (ECTs)"
 description: |-
-  Find out about the support early career teachers get when they start teaching, including mentor support and training based on the early career framework (ECF).
+  Find out about the support early career teachers get when they start teaching, including mentor support and training based on the initial teacher training early career framework (ITTECF).
 promo_content:
   - "content/shared/block-promos/mailing_adviser_routes"
 ---
 
 You’ll be known as an early career teacher (ECT) for your first 2 years in teaching. The term early career teacher (ECT) has replaced newly qualified teacher (NQT).
 
-During your 2 year induction, you’ll receive a package of training and support based on the [early career framework (ECF)](https://www.gov.uk/guidance/how-the-early-career-framework-ecf-supports-induction). This is designed to help you build the skills and knowledge you need to feel confident and empowered in your classroom.
+During your 2-year induction, you’ll receive a package of training and support based on the [initial teacher training and early career framework (ITTECF)](https://www.gov.uk/government/publications/initial-teacher-training-and-early-career-framework?). Your package of training and support is referred to as the early career teacher entitlement (ECTE).
 
-This won’t impact your pay or progression opportunities. You’ll still be able to progress on the pay scale during your induction period.
+The ECTE is designed to help you build the skills and knowledge you need to feel confident and empowered in your classroom. It won’t impact your pay or progression opportunities. You’ll still be able to progress on the pay scale during your induction period.
 
 If your teacher training course does not lead to English qualified teacher status (QTS) (for example, if your course only leads to a PGCE or other academic award) you will not receive the following support.
 
-## ECF-based training
+### Your early career teacher training programme
 
-You’ll receive training based on the early career framework (ECF) as part of your induction period. This is called ECF-based training.
+You’ll receive training based on the initial teacher training and early career framework (ITTECF) as part of your early career teacher entitlement (ECTE). This training provides a bridge between your initial teacher training (ITT) and your teaching career. Your ITT helps you qualify as a teacher, and your ECTE helps you further develop your teaching practice and working habits.
```

Comment:
I find this para a bit awkward to read as it has so many acronyms in it. Is it worth splitting it into 2 paras eg `You'll receive.....` and 'This training.....`